crypto3/libs/blueprint/include/nil/blueprint/bbf/bbf_wrapper.hpp

```diff
@@ -216,7 +216,14 @@ namespace nil {
                 using component_type = plonk_bbf_wrapper<BlueprintFieldType>;
                 using var = typename component_type::var;
 
-                context_type ct = context_type(assignment,8,start_row_index); // max_rows = 8
+                // get_description() function returns the actual number of rows used. In order to allow more rows to be used we
+                // must add the max_rows to the returned sizes.
+                std::size_t max_rows = 8;
+                auto desc = assignment.get_description();
+                desc.usable_rows_amount += max_rows;
```

Comment:
at assignment-genereation stage we don't do anything like that, so we end up with two allocation_logs of different sizes, which is probably not so good.

Reply:
Changed this. Now the caller needs to know, that the description provided is actually the used part.